cmd/zpool/zpool_main.c

```diff
@@ -2427,21 +2427,37 @@ zpool_do_import(int argc, char **argv)
 	boolean_t do_rewind = B_FALSE;
 	boolean_t xtreme_rewind = B_FALSE;
 	boolean_t do_scan = B_FALSE;
+	boolean_t do_path = B_FALSE;
 	uint64_t pool_state, txg = -1ULL;
 	char *cachefile = NULL;
 	importargs_t idata = { 0 };
 	char *endptr;
 
 	/* check options */
-	while ((c = getopt(argc, argv, ":aCc:d:DEfFmnNo:R:stT:VX")) != -1) {
+	while ((c = getopt(argc, argv, ":aCc:P:d:DEfFmnNo:R:stT:VX")) != -1) {
```

Comment:
`P` should be described in the comment above this function and added to the `zpool(8)` man page.  `-P` is good choice for consistency with the `zpool add|iostat|list|split|status -P` option.